extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessState.scala

```diff
@@ -89,13 +89,15 @@ object ProcessActionState extends Enumeration {
 
 object StateStatus {
   type StatusName = String
-  implicit val configuration: Configuration = Configuration
-    .default
-    .withDefaults
-    .withDiscriminator("type")
+
+  // Temporary encoder/decoder
```

Comment:
Let's write that  `StateStatus` has to have `Decoder` defined because it is accidentally decoded along with `ProcessState` in the migration process (`StandardRemoteEnvironment` class). In all cases (this one and for FE purposes) only info about the status name is essential.

Reply:
Fixed